ArduCopter/AP_Arming.cpp

```diff
@@ -159,6 +159,10 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
                 return false;
             }
         }
+        if (copter.g.failsafe_gcs == FS_GCS_ENABLED_CONTINUE_MISSION) {
+            // FS_GCS_ENABLE == 2 is deprecated
+            check_failed(ARMING_CHECK_PARAMETERS, display_failure, "FS_GCS_ENABLE=2 deprecated, see FS_OPTIONS");
```

Comment:
change "deprecated" to "removed"

Reply:
should also fix in param description to be removed

Reply:
replaced "deprecated" with "removed" in param descriptions for FS_GCS_ENABLE and FS_THR_ENABLE